File: In-App Settings Panel/app/src/main/java/com/eypcnn/qsettingspanel/MainActivity.java
package com.eypcnn.qsettingspanel;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.provider.Settings;
import android.view.View;
import android.widget.Button;

import java.util.Set;

public class MainActivity extends AppCompatActivity {

    Button internet, volume, nfc;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        internet = findViewById(R.id.internetButton);
        volume = findViewById(R.id.volumeButton);
        nfc = findViewById(R.id.nfcButton);

        internet.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(Settings.Panel.ACTION_INTERNET_CONNECTIVITY));
            }
        });

        volume.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(Settings.Panel.ACTION_VOLUME));
            }
        });

        nfc.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(Settings.Panel.ACTION_NFC));
            }
        });
    }
}

File: In-App Settings Panel Kotlin/app/src/main/java/com/eypcnn/qsettingspanel/MainActivity.kt
package com.eypcnn.qsettingspanel

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.provider.Settings
import android.view.View
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val internet = findViewById<Button>(R.id.internetButton)
        val volume = findViewById<Button>(R.id.volumeButton)
        val nfc = findViewById<Button>(R.id.nfcButton)

        internet.setOnClickListener(View.OnClickListener { startActivity(Intent(Settings.Panel.ACTION_INTERNET_CONNECTIVITY)) })
        volume.setOnClickListener(View.OnClickListener { startActivity(Intent(Settings.Panel.ACTION_VOLUME)) })
        nfc.setOnClickListener(View.OnClickListener { startActivity(Intent(Settings.Panel.ACTION_NFC)) })

    }
}